### ISSUE TITLE
cannot load such file -- simplecov (LoadError)

### ISSUE THREAD

**Issue by oniofchaos** [codeclimate/ruby-test-reporter#154]
When I run `codeclimate-test-reporter` on the console, I get this error. Am I missing something?

```bash
/Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/lib/code_climate/test_reporter/formatter.rb:7:in `require': cannot load such file -- simplecov (LoadError)
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/lib/code_climate/test_reporter/formatter.rb:7:in `<top (required)>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/lib/codeclimate-test-reporter.rb:7:in `require'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/lib/codeclimate-test-reporter.rb:7:in `<top (required)>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/bin/codeclimate-test-reporter:3:in `require'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/gems/codeclimate-test-reporter-1.0.0/bin/codeclimate-test-reporter:3:in `<top (required)>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/bin/codeclimate-test-reporter:22:in `load'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/bin/codeclimate-test-reporter:22:in `<main>'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/dillonwelch/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by maxjacobson** [comment]
@oniofchaos Thanks for the issue and apologies for the inconvenience. I believe we fixed this in v1.0.3. Will you try updating?

**Comment by oniofchaos** [comment]
Not sure what I did wrong, but after I unlocked the gem version it worked. Thanks
